react-events/my-app/src/App.js

```diff
@@ -0,0 +1,26 @@
+import logo from './logo.svg';
+import './App.css';
+import CustomButton from './CustomButton';
+
+function App() {
+  function handleCustomClick(text) {
+  alert(text);
+}
+  return (
+    <div className="App">
+      <header className="App-header">
+        <img src={logo} className="App-logo" alt="logo" />
+        <p>
+          Edit <code>src/App.js</code> and save to reload.
+        </p>
+      <CustomButton color='red' text='Pretty Cool Red' onCustomClick={handleCustomClick} />
```

Comment:
It's customary to use double quotes (`"`) in JSX props. I think it's to make it look more like HTML.